src/serverMain/resources/userFrontend/userFrontend.js

```diff
@@ -87,7 +85,7 @@ function normalStartup() {
               // Success
               setDatetimeElement(Date.now())
               window.localStorage.setItem("email", email);
-              window.localStorage.setItem("checkin", `${btoa(email)}::${locationId}::${Date.now().toString()}`);
+              window.localStorage.setItem("checkin-" + locationId, `${btoa(email)}::${Date.now().toString()}`);
```

Comment:
Good catch!

Reply:
Thanks ... But what do you mean by that 😅

Reply:
That you could have 2 concurrent checkins in 2 locations